stdlib/public/stubs/Assert.cpp

```diff
@@ -33,7 +63,8 @@ _swift_stdlib_reportFatalErrorInFile(const char *prefix, intptr_t prefixLength,
                                    uintptr_t line,
                                      uint32_t flags) {
   char *log;
-  asprintf(&log, "%.*s: %.*s%sfile %.*s, line %zu\n", (int)prefixLength, prefix,
+  swift_asprintf(&log, "%.*s: %.*s%sfile %.*s, line %zu\n",
+           (int)prefixLength, prefix,
```

Comment:
Align parameters. The addition of "swift_" shifted things over.

Look into using git-clang-format on your changes. I am not sure of a quick guide of how to get it up and running, worked it out on my own but didn't take notes.